include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -728,6 +728,7 @@ def Torch_ConstantIntOp : Torch_Op<"constant.int", [
   // Indicate that the operation has a custom parser and printer method.
   let hasCustomAssemblyFormat = 1;
   let hasFolder = 1;
+  let hasVerifier = 1;
```

Comment:
I think if you change `AnyI64Attr` to `I64Attr` above then you won't need the custom verifier.